model_compression_toolkit/target_platform_capabilities/target_platform/target_platform_model.py

```diff
@@ -223,3 +223,12 @@ def show(self):
 
         """
         pprint.pprint(self.get_info(), sort_dicts=False)
+
+    def set_quantization_format(self,
+                                quantization_format: Any):
+        """
+        Set quantization format.
+        Args:
+            quantization_format: quantization_format from quantization format enum.
```

Comment:
Maybe you can write a bit more like: "A quantization format (fake-quant, int8m etc.) from enum QuantizationFormat"